project/include/ui/JoystickEvent.h

```diff
@@ -13,7 +13,7 @@ namespace lime {
 
 		JOYSTICK_AXIS_MOVE,
 		JOYSTICK_HAT_MOVE,
-		JOYSTICK_TRACKBALL_MOVE,
+		// JOYSTICK_TRACKBALL_MOVE,
```

Comment:
Any reason for commenting this, rather than deleting?

Reply:
Fixed by setting the enum values to constants so existing applications don't break.